rust/src/dcerpc/mod.rs

```diff
@@ -17,6 +17,6 @@
 
 pub mod dcerpc;
 pub mod dcerpc_udp;
-pub mod parser;
 pub mod detect;
 pub mod log;
+pub mod parser;
```

Comment:
are these rustfmt changes?

Reply:
Yes, by default it will re-order imports alphabetically: https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=imports#reorder_imports